scenes/autoload/TransitionController.gd

```diff
@@ -83,6 +83,12 @@ func _on_animation_player_animation_finished(anim_name):
 	if anim_name == "Fade Out Scene":
 		if next_scene != null:
 
+			# Stop level timer
+			TimeTrialController.stop_timer()
```

Comment:
Is this the right place to stop the time? Maybe when they initially reach the door?